formatTest/unit_tests/expected_output/modules.re

```diff
@@ -515,24 +516,42 @@ module M = {
 
 module N = {
   open M;
-  let z = M.(34);
+  let z = {
+    M.(34);
```

Comment:
Why does this local open sugar have braces?

Reply:
This is how the original input file looked.